src/nft/components/profile/view/WalletAssetDisplay.tsx

```diff
@@ -100,7 +100,7 @@ export const WalletAssetDisplay = ({ asset, isSellMode }: { asset: WalletAsset;
                           borderRadius="4"
                           marginLeft="4"
                           objectFit="cover"
-                          src={`/nft/svgs/marketplaces/${market.toLowerCase()}.svg`}
+                          src={`/nft/svgs/marketplaces/${market?.toLowerCase()}.svg`}
```

Comment:
if the market is unavailable should we not have a src at all?

Reply:
good call